content/en/docs/ops/configuration/security/migrate-alpha-policy/index.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Migrate the deprecated alpha security policy
+description: A tutorial to help customers migrate the deprecated alpha security policy to the beta version.
+keywords: [security,policy,migrate,alpha,beta,deprecate,peer,jwt,authorization]
+weight: 20
+owner: istio/wg-security-maintainers
+test: n/a
+---
+
+Istio beta security policy (`PeerAuthentication`, `RequestAuthentication` and `AuthorizationPolicy`) has been released
+since release 1.5 and the alpha security policy (`MeshPolicy`, `Policy`, `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding`)
+has been deprecated and no longer supported starting with release 1.6.
+
+It is required to migrate the alpha security policy to the beta version before upgrading to Istio 1.6 and later.
+This tutorial helps customers who are still using the alpha security policy in a `pre-1.6` version to migrate to
+the beta security policy and unblock the upgrade to Istio 1.6 and later.
+
+## Overview
+
+The control plane should be upgraded first to bring in the support of `v1beta1` security policy. It is recommended to first
+upgrade to Istio 1.5 as a transitive version because it supports both `v1alpha1` and `v1beta1` security policy. You complete
```

Comment:
```suggestion
upgrade to Istio 1.5 as a transitive version because it supports both `v1alpha1` and `v1beta1` security policies. You complete
```
```suggestion
upgrade to Istio 1.5 as a transitive version because it supports both `v1alpha1` and `v1beta1` security policy. You complete
```